content/applications/finance/accounting/fiscal_localizations/localizations/italy.rst

```diff
@@ -2,78 +2,382 @@
+
+It usually is a **summary invoice** containing a list of multiple sales of goods or services,
+carried out in the month. The business is allowed to **group** the sales into **one invoice**,
+generally issued at the **end of the month** for accounting purposes. Deferred invoices are default
+for **wholesaler** having recurrent clients.
+
+If the goods are transported by a **carrier**, every delivery has an associated **Documento di
+Transporto (DDT)**, or **Transport Document**. The deferred invoice **must** indicate the details of
+all the **DDTs** information for better tracing.
+
+.. note::
+   e-Invoicing of deferred invoices requires the `l10n_it_stock_ddt` :ref:`module <italy/modules>`.
+   In this case, a dedicated :guilabel:`Tipo Documento` `TD24` is used in the e-Invoice.
+
+Odoo exports moves as `TD24` if the following conditions are met:
+
+- Is an invoice;
+- Is associated to deliveries whose **DDTs** have a **different** date than the issuance date of the
+  invoice.
+
+Vendor bills
+============
+
+When creating a **vendor bill**, the :guilabel:`FatturaPA` feature is enabled by default and
+automatically activated under :menuselection:`Accounting --> Configuration --> Journals --> Vendor
+Bills (journal) --> Advanced Settings (tab) --> Electronic Data Interchange`.
+
+When inserting taxes in a vendor bill, you have the possibility to choose **Reverse Charge** taxes,
+these are automatically activated in the Italian fiscal position. By going to
+:menuselection:`Configuration --> Taxes`, you will see that the 10% and 22% services and goods taxes
+are activated and have the correct tax grids; these are set up automatically to ensure the correct
+booking of accounting entries and display of the tax report.
+Once the bill is confirmed, a banner appears, and the bill can be sent to the EDI service.
+
+San Marino
+==========
+
+Invoices
+--------
+
+San Marino and Italy have special agreements on e-Invoicing operations. As such, **invoices** follow
+the regular **reverse charge** rules. Additional requirements are not enforced by Odoo, however, the
+user is requested by the **State** to:
+
+- Select a tax with the option :guilabel:`Has exoneration of tax (Italy)` ticked, and the
+  :guilabel:`Exoneration` set to `N3.3`;
+- Use the generic :abbr:`SdI (Systema di Interscambio)` :guilabel:`Codice Destinatario` `2R4GT08`.
+  The invoice is then routed by a dedicated office in San Marino to the correct business.
+
+Bills
+-----
+
+When a **paper bill** is received from San Marino, any Italian company **must** submit that invoice
+to the **Agenzia delle Estrate** by indicating the e-Invoice's :guilabel:`Typo Documento` field with
+the special value `TD28`.
 
-Receive invoices
-~~~~~~~~~~~~~~~~
+Odoo exports a move as `TD28` if the following conditions are met:
 
-This process is ran automatically once a day.
+- Is a vendor bill;
+- At least one tax on the invoice lines targets the tax grids `VJ`;
```

Comment:
Tax grids `VJ` again. Maybe we can have a little note about it, and link it on the occurrences? What's the default practice for repetitive info?

Reply:
I added a H3 heading and section on tax grids, that way the info is centralized much more easily